### PR TITLE
Added cheatsheet for Indian Helpline Numbers

### PR DESCRIPTION
**Pull request by aiyankovil**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Indian Helpline numbers Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

added cheat sheet for Indian helpline numbers

---
https://duck.co/ia/view/indian_helpline_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@aiyankovil Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by sahildua2305** [comment]
@aiyankovil As you can see, your cheat sheet is failing on [Travis Build](https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/138584938#L773), you should name your main json file as `indian-helpline.json`

**Comment by aiyankovil** [comment]
@sahildua2305  sir,trying to change the file name. thks for the help

**Comment by aiyankovil** [comment]
@sahildua2305 sir should it be indian_helpline.json or indian-helpline.json

**Comment by sahildua2305** [comment]
@aiyankovil It should be indian-helpline.json if `id = "indian_helpline_cheat_sheet"

**Comment by aiyankovil** [comment]
@sahildua2305 the specified changes has been changed sir.

**Comment by sahildua2305** [comment]
@aiyankovil And your pull request now passes all tests! :tada:

**Comment by aiyankovil** [comment]
@sahildua2305 thks

**Comment by aiyankovil** [comment]
@sahildua2305 sir ,there has been branch conflicts.how can i resolve that

**Comment by sahildua2305** [comment]
@aarohmankad Check this link
http://blog.michelemattioni.me/2013/01/29/what-do-i-do-when-my-pull-request-does-not-merge-automatically-in-master/

Let me know if you are still stuck.

File: share/goodie/cheat_sheets/json/indian-helpline.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+         },
+         {  
+            "key":"1094",
+            "val":"Missing Children and Women (Delhi)"
+         },
+         {  
+            "key":"1096",
+            "val":"Crime Against Women (Delhi)"
+         },
+         {
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/indian-helpline.json]: I think we should separate this into two sections

**Comment by aiyankovil** [comment]
@sahildua2305 i am still stuck with the conflicts .what should i do.it affected my other pull requests also

File: share/goodie/cheat_sheets/json/indian-helpline.json (hunk 1 above)
Comment (aiyankovil) [inline share/goodie/cheat_sheets/json/indian-helpline.json]: @gautamkrishnar i will make the the specified changes . but now i am stuck with these conflicts which stops my pull request from deploying

**Comment by gautamkrishnar** [comment]
@aiyankovil lets execute the following commands:

cd zeroclickinfo-goodies

**Comment by edgesince84** [comment]
@aiyankovil I'll get some help from internal staff for you too -- if @sahildua2305 and @gautamkrishnar don't get to it before we do! 😆

**Comment by sahildua2305** [comment]
@aiyankovil I have figured out the cause of conflicts.
You had previously added `un.json` [here](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3285/commits/ca33889bc51fe4b1a538babfced47e7fc3cfc8fb) and then deleted the file [here](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3285/commits/1bcc6ae2322f67bc0368ed46b40ac86271117897). And now that your another PR has been merged, duckduckgo:master branch already has that file `un.json`.

Can you please create another PR from another branch which has only one file `indian-helpline.json` ?

**Comment by moollaza** [comment]
@sahildua2305 is correct. A new PR is the simplest approach here, from a clean branch that's up to date.

Something like this should work...

```
git checkout master
git pull upstream/master
git checkout -b indian-helpline-cheatsheet
** add your JSON file **
git status (everything looks good?)
git commit -am "New Indian Helpline cheat sheet"
git push
** open a new pull request **
```

**Comment by aiyankovil** [comment]
@moollaza @sahildua2305 should i create new ia page also

**Comment by aiyankovil** [comment]
@moollaza can it be merged with this conflicts.

**Comment by aiyankovil** [comment]
@moollaza @sahildua2305  i have tried the above commands but still facing the same problem

**Comment by tagawa** [comment]
Hi @aiyankovil. No, you don't need to create a new IA page.

From the GitHub help comment above, it looks like this could fix the merge conflict:

Step 1: From your project repository, check out a new branch and test the changes.

`git checkout -b aiyankovil-inhpl master`
`curl -L https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3285.patch | git am -3`

Step 2: Merge the changes and update on GitHub.

`git checkout master`
`git merge --no-ff aiyankovil-inhpl`
`git push origin master`

**Comment by moollaza** [comment]
> i have tried the above commands but still facing the same problem

@aiyankovil do you have an `upstream` remote pointing to `duckduckgo/zeroclickinfo-goodies`?

What does `git remote -v` show?

**Comment by gitcodes** [comment]
@aiyankovil I prefer you make create a new branch and json file with different name and have a new PR.

**Comment by tagawa** [comment]
Hi @aiyankovil. As @gitcodes suggested, would you like to create a new branch (e.g. `git checkout -b helpline`), copy your JSON file into that and send a new pull request? We could then close this one.

**Comment by aiyankovil** [comment]
@moollaza origin  https://github.com/aiyankovil/zeroclickinfo-goodies.git (fetch)                                                                                                                       
origin  https://github.com/aiyankovil/zeroclickinfo-goodies.git (push)

**Comment by moollaza** [comment]
@aiyankovil that's your problem.

You need to `git remote add upstream git@github.com:duckduckgo/zeroclickinfo-goodies.git`

Then you will be able to pull down the code from our repo to keep your fork up to date

**Comment by moollaza** [comment]
Either way, it'll be best to work from a new branch and pull request. Closing this one 👍

**Comment by adityasiwan** [comment]
@moollaza  @aiyankovil I would like to take it over and complete it.
@moollaza May you please delete this IA page so that i can create a new one. :smile:

**Comment by zekiel** [comment]
@adityasiwan I took care of that just now so should be OK for you to proceed.

**Comment by adityasiwan** [comment]
@zekiel Getting an error here while creating new IA.
screenshot 58

**Comment by aiyankovil** [comment]
@adityasiwan u can proceed. I will do the necessary

**Pull request closed without merging**